WTWSMS/history/characters/adnanite.txt

```diff
@@ -1,39 +1,9 @@
-33900 = {
```

Comment:
Why all the removals here?

Reply:
There are all vanilla duplicates of Enlil's characters.

Reply:
Is 36504 the Enlil version of this character?

Reply:
Yes.